React/Base/RCTConvert.h

```diff
@@ -18,6 +18,21 @@
 #import "RCTLog.h"
 #import "RCTPointerEvents.h"
 
+typedef NS_ENUM(NSInteger, RCTBarStyle) {
```

Comment:
Can you add a comment here that these should mirror UIBarStyle  and UIDataDetectorTypes? We don't want to extend these with unsupported keys.